index.html

```diff
@@ -5,8 +5,70 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet">
 </head>
+
 <body>
-    
+        
+    <header class="header">
+        <h1>Weather Report</h1>
+        <h2>The weather for...
+            <span id="headerCityName" class="header__city-name"></span>
+        </h2>
+    </header>
+    <section class="content">
+
+        <div class="wrapper">
+        <div class="box1">
+            <h1>Temperature:</h1>
+            <h1><span id="temp"></span></h1>
+            <div class="temperature__content">
+                <div class="temperature__controls">
+                    <span id="incTemp">⬆️</span>
+                    <span id="decTemp">⬇️</span>
+                    <div>Use the arrows to adjust the temperature.</div>
+                </div>
+                
+            </div>
+        </div>
+        <div class="box2">
+            <h1>Today's Horoscope</h1>
+            <li><a href= "https://www.horoscope.com/us/horoscopes/general/index-horoscope-general-daily.aspx">My horoscope</a></li>
+        </div>
+            
+        <div class="box3">
+            <section class="weather_garden">
+                <h1>Weather Garden</h1>
+                <div id="gardenStuff" class="garden__stuff">
+                    <div id="sky"></div><br><br><br><br>
+                    <div id="landscape"></div><br><br>
+                    <head> What time is it in EST?</head>
+                    <body>
```

Comment:
The body tag defines the document's body. You wouldn't use it here. You could use the p tag without it.